src/main/command/other/help.py

```diff
@@ -0,0 +1,102 @@
+import discord
+
+import main
+from command.command import Command, Parameter, Category
+
+
+class Help(Command):
+    def __init__(self):
+        super().__init__("help", "Shows a command info or the list of the commands", ["commands", "?"], [
+            Parameter("command")], category=Category.GENERAL)
+
+    # noinspection PyMethodParameters
+    async def execute(self, message: discord.Message, parameters: str, client: discord.Client):
+        command_list = Command.__subclasses__()
+        command_list.sort(key=main.sort_commands)
+
+        if not parameters or parameters == '':
```

Comment:
```suggestion
        if not parameters:
```
Empty string is false in python